generator/python/build.gradle

```diff
@@ -49,39 +50,41 @@ task generateClient(type: GradleBuild) {
         
         def clientOutputDirPerVersion = clientOutputDir + "/" + parsedApiName;
         def specPath = it.path;
-        def packageBase =  criteoApiVersion; 
-        if (packageBase.matches("[0-9].*"))
-             packageBase= "v" + packageBase;
+        def versionCategory = criteoApiVersion == 'preview' ? 'preview' : 'stable'
+  
+        def versionInNamespace =  criteoApiVersion
+        if (versionInNamespace.matches("[0-9].*"))
+             versionInNamespace= "v" + versionInNamespace
 
-        packageBase = "${criteoPackage}.${criteoService}.${packageBase}".replace('.', nameSeparator).toString(); // example : criteo_api_marketingsolutions_v2021_10
-        def packageVersion = "${criteoApiVersion}.${generatorVersion}.${getFormattedDate()}"; // example : 2021_10.1.211110
-
-        def generateTask = task("openApiGenerate_python_" + parsedApiName, type: GenerateTask.class) {
+        def namespaceBase = "${criteoPackage}.${criteoService}.${versionInNamespace}".replace('.', nameSeparator).toString(); // example: criteo\api\marketingsolutions\v2021_10
```

Comment:
The comment seems wrong, should be criteo_api_marketingsolutions_v2021_10 right?

Reply:
Done